config/projects.yml

```diff
@@ -269,12 +269,12 @@ intl-bundle:
       php: ['7.1']
       versions:
         symfony: ['2.8', '3.1', '3.2']
-        sonata_user: ['2', '3']
+        sonata_user: ['3']
     2.x:
       php: ['5.3', '5.4', '5.5', '5.6', '7.0', '7.1']
       versions:
         symfony: ['2.3', '2.7', '2.8', '3.1', '3.2']
-        sonata_user: ['2', '3']
```

Comment:
We don't support 2.x anymore. IMHO we should risk this break. It looks like a mistake we've done in the past.